src/artm/core/data_loader.cc

```diff
@@ -102,8 +102,11 @@ LocalDataLoader::~LocalDataLoader() {
   }
 }
 
-void LocalDataLoader::AddBatch(const Batch& batch, bool invoke) {
+bool LocalDataLoader::AddBatch(const AddBatchArgs& args, bool invoke) {
```

Comment:
Could you try removing the 'bool invoke' argument?
I think you can get the same value from MasterComponentConfig.online_processing_mode